prow/test/integration/prow/config.yaml

```diff
@@ -16,3 +16,24 @@ prowjob_default_entries:
     repo: 'testRepo'
     config:
       tenant_id: 'tester'
+
+gerrit:
+  tick_interval: 30s
```

Comment:
can this be shorter so that the test doesn't need to sleep for 30s?

Reply:
Done.